<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GA - NSGA-II</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="algorithmProgressBar">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>50</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="gaPushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run GA Algorithm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nsga2PushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run NSGA-II Algorithm</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeElapsedLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time elapsed :</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeElapsed">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Search_Params"/>
    <addaction name="action_Load_Graph"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Display">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuDock_Widget">
     <property name="title">
      <string>Dock&amp;Widget</string>
     </property>
    </widget>
    <addaction name="menuDock_Widget"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Display"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Search_Params">
   <property name="text">
    <string>&amp;Search Params</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Configuration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Configuration</string>
   </property>
  </action>
  <action name="action_Solutions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Solutions</string>
   </property>
  </action>
  <action name="action_Load_Graph">
   <property name="text">
    <string>&amp;Load Graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G, Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
